libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts

```diff
@@ -108,22 +108,37 @@ export class FinanceResolver {
       input.listPath,
       user,
     )
+
+    this.auditService.audit({
+      auth: user,
+      namespace: '@island.is/api/finance',
+      action: 'getDocumentList',
+      resources: [input.listPath, input.dayFrom, input.dayTo],
+    })
+
```

Comment:
What is the reason to move from the decorator?
The decorator can be updated to log the resources, it logs the authentication be default and uses the namespace from the decorator on the controller.
I personally prefer the decorator over the service when I can.

Reply:
Should add these as meta properties instead. The resources field is designed for specific resource ids which you want to query all audit entries for.